frontend/src/features/comparisons/ComparisonList.tsx

```diff
@@ -5,10 +5,23 @@ import { Compare as CompareIcon } from '@mui/icons-material';
 
 import type { Comparison } from 'src/services/openapi';
 import VideoCard from '../videos/VideoCard';
+import { VideoObject } from 'src/utils/types';
+import { videoFromRelatedEntity } from 'src/utils/entity';
 
 const ComparisonThumbnail = ({ comparison }: { comparison: Comparison }) => {
   const { t } = useTranslation();
   const { entity_a, entity_b } = comparison;
+
+  const videoA: VideoObject = {
+    ...videoFromRelatedEntity(entity_a),
+    criteria_scores: comparison.criteria_scores,
+  };
+
+  const videoB: VideoObject = {
+    ...videoFromRelatedEntity(entity_b),
+    criteria_scores: comparison.criteria_scores,
+  };
```

Comment:
It does not seem correct to associate the comparison scores to each video. And `criteria_scores` is optional in `VideoObject`, so why not omitting it?

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/623/commits/af3f37f462455430345dce0277a9aee71f588a57.